backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/config/ConfigValues.java

```diff
@@ -1415,6 +1415,9 @@
     @OptionBehaviourAttribute(behaviour = OptionBehaviour.CommaSeparatedStringArray)
     HostDevicePassthroughCapabilities(ClientAccessLevel.Admin),
 
+    @TypeConverterAttribute(String.class)
```

Comment:
how about replacing this with:
@TypeConverterAttribute(List.class)
@OptionBehaviourAttribute(behaviour = OptionBehaviour.CommaSeparatedStringArray)